index.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+      <meta charset="utf-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>JS Documentation</title>
+      <link rel="stylesheet" href="styles.css">
+        </head>
+    <body>
+
+      <nav id="navbar" class="flex-container">
+        <header>MENU</header>
+        <div class="flex-container">
+          <a class="nav-link" href="#1">Intro</a>
+          <a class="nav-link" href="#2">What you should already know</a>
+          <a class="nav-link" href="#3">JavaScript and Java</a>
+          <a class="nav-link" href="#4">Variables</a>
+          <a class="nav-link" href="#5">Declaring variables</a>
+        </div>
+      </nav>
+
+      <main id ="main-doc">
+
+        <section class="main-section">
+          <a href="#5"><img src="https://pic.onlinewebfonts.com/svg/img_116266.png" alt="Arrow Down"></a>
+        </section>
+
+        <section id="1" class="main-section">
```

Comment:
Please give meaningful id names. This will help you and others to easily add new features or debug your code.